lib/client.js

```diff
@@ -75,6 +75,16 @@ class Client {
     return url;
   }
 
+  /**
+   * Set the axios Authorization header to `Bearer ${token}`.
+   *
+   * @param {string} token The access token value.
+   */
+  // eslint-disable-next-line class-methods-use-this
+  set accessToken(token) {
+    axios.defaults.headers.common.Authorization = `Bearer ${token}`;
```

Comment:
Not sure about this -- with ^this implementation, are the headers scoped *only* to our fhir client instance?

Either way: the same problem exists with the standalone `httpGet` function that is closed-over.  (which also resets the "Accept" header on every invocation)